src/Data/Fixed.purs

```diff
@@ -79,6 +94,25 @@ instance knownPrecisionOne :: KnownPrecision One where
 instance knownPrecisionTenTimes :: KnownPrecision p => KnownPrecision (TenTimes p) where
   reflectPrecision _ = 10 * reflectPrecision (PProxy :: PProxy p)
 
+-- | Get the number of decimal places associated with a given `Precision` at
+-- | the value level.
+-- |
+-- | ```
+-- | > reflectPrecisionDecimalPlaces (PProxy :: PProxy P1000)
+-- | 3
+-- | ```
+reflectPrecisionDecimalPlaces
+  :: forall precision
+   . KnownPrecision precision
+  => PProxy precision
+  -> Int
+reflectPrecisionDecimalPlaces _ =
```

Comment:
What about adding this to the `KnownPrecision` class?

Reply:
Actually, this makes me wonder what we'd do with this if we ever implement non-decimal precisions, as in #1.

Reply:
I think this would still work? Might need a minor adjustment (perhaps we want `Int.ceil` instead of `Int.round` or something), but it still makes sense to ask for the (maximum) number of decimal places you need to be able to display a number to that precision, I think.

Reply:
Yeah, makes sense.

Reply:
I've just realised it won't actually continue to work; if the denominator has any prime factors other than 2 or 5 in its simplest form, then the decimal representation will be non-terminating. I'll add a comment on #1.